client/src/i18n/en-US.json

```diff
@@ -755,7 +755,7 @@
       },
       "reply": {
         "ariaLabel": "Comment Reply.  Author {username}.",
-        "referenceButtonAria": "Jump to referenced comment",
+        "referenceButtonAria": "Jump to referenced comment.",
```

Comment:
This also makes the mistake of using periods as ending punctuation for an `aria-label`.

Reply:
See above.

Reply:
I removed the period for this.